includes/Core/Util/Migration_1_45_0.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Migration for 1.45.0
```

Comment:
I think it should be `1.46.0`, shouldn't it? The whole file and the class name should be renamed to `Migration_1_46_0`.

Reply:
https://github.com/google/site-kit-wp/pull/4305/commits/95aec903472e378b824d99b9e3a98f9b390d8a3b